featurebyte/service/feature_materialize.py

```diff
@@ -44,6 +46,8 @@
 from featurebyte.service.session_manager import SessionManagerService
 from featurebyte.session.base import BaseSession
 
+OFFLINE_STORE_TABLE_REDIS_LOCK_TIMEOUT_SECONDS = 3600
```

Comment:
When populating entity lookup table (for item table), it can takes more than one hour (local testing takes about 2-3 hours). We may need to increase this number.

Reply:
Updated the logic to reduce the scope of lock to offline store table writes only, excluding online store materialize. With that I think the current timeout should be sufficient. The slow part is online store update materialize we should further investigate.